buildSrc/src/main/resources/uber-plugin-descriptor.properties

```diff
@@ -0,0 +1,25 @@
+# Elasticsearch uber plugin descriptor file
+# This file must exist as 'uber-plugin-descriptor.properties' in a folder named `elasticsearch`.
```

Comment:
I think better name than "uber" would be "meta" (as in meta physical, it isn't really a plugin)